msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2023-12-27 10:21-0600\n"
"PO-Revision-Date: 2023-12-27 17:27\n"
"Last-Translator: \n"
"Language-Team: French\n"
"Language: fr_FR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
"X-Crowdin-Project: duplicity\n"
"X-Crowdin-Project-ID: 539508\n"
"X-Crowdin-Language: fr\n"
"X-Crowdin-File: /main/po/duplicity.pot\n"
"X-Crowdin-File-ID: 6\n"

#: ../bin/duplicity:76
msgid "INT intercepted...exiting."
msgstr "INT intercepté... sortie"

#: ../bin/duplicity:84
#, python-format
msgid "GPG error detail: %s"
msgstr "Détails de l’erreur GPG : %s"

#: ../bin/duplicity:91
#, python-format
msgid "User error detail: %s"
msgstr "Détails de l’erreur utilisateur : %s"

#: ../bin/duplicity:98
#, python-format
msgid "Backend error detail: %s"
msgstr "Détails de l’erreur de sauvegarde : %s"

#: ../duplicity/tempdir.py:157
#, python-format
msgid "Using temporary directory %s"
msgstr "Utilisation du dossier temporaire %s"

#: ../duplicity/tempdir.py:201
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "Enregistrement (mktemp) du fichier temporaire %s"

#: ../duplicity/tempdir.py:227
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "Enregistrement (mkstemp) du fichier temporaire %s"

#: ../duplicity/tempdir.py:259
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Oubli du fichier temporaire %s"

#: ../duplicity/tempdir.py:262
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr "Tente d'oublier le fichier temporaire inconnu %s - ceci est probablement un bogue."

#: ../duplicity/tempdir.py:281
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Suppression du fichier temporaire toujours en mémoire %s"

#: ../duplicity/tempdir.py:284
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "Le nettoyage du fichier temporaire %s a échoué"

#: ../duplicity/tempdir.py:290
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr "Le nettoyage du dossier temporaire %s a échoué - ceci est probablement un bogue."

#: ../duplicity/backend.py:92
#, python-format
msgid "Import of %s %s"
msgstr "Importation de %s %s"

#: ../duplicity/backend.py:197
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Impossible d'initialiser le moteur : %s"

#: ../duplicity/backend.py:388
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Trace inverse de l'erreur précédente : %s"

#: ../duplicity/backend.py:407
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Abandonner après %s tentatives. %s: %s"

#: ../duplicity/backend.py:413
#, python-format
msgid "Attempt of %s Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backend.py:512
#, python-format
msgid "Reading results of '%s'"
msgstr "Lecture des résultats de « %s »"

#: ../duplicity/backend.py:539
#, python-format
msgid "Writing %s"
msgstr "Écriture de %s"

#: ../duplicity/backend.py:580
#, python-format
msgid "File %s not found locally after get from backend"
msgstr ""

#: ../duplicity/manifest.py:95
#, python-format
msgid "Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr "Erreur fatale : l'hôte de la source de sauvegarde a changé.\n"
"Nom d'hôte courant : %s\n"
"Nom d'hôte précédent : %s"

#: ../duplicity/manifest.py:102
msgid "Fatal Error: Backup source directory has changed.\n"
msgstr ""

#: ../duplicity/manifest.py:115
msgid "Aborting because you may have accidentally tried to backup two different data sets to the same remote location, or using the same archive directory.  If this is not a mistake, use the --allow-source-mismatch switch to avoid seeing this message"
msgstr "Annulation parce que vous avez peut-être accidentellement essayé de sauvegarder deux jeux de données différents au même emplacement distant ou utilisé le même dossier d'archives. Si ceci est une erreur, utilisez l'option --allow-source-mismatch pour éviter de voir ce message."

#: ../duplicity/manifest.py:221
#, python-format
msgid "Found manifest volume %s"
msgstr ""

#: ../duplicity/manifest.py:228
#, python-format
msgid "Found %s volumes in manifest"
msgstr ""

#: ../duplicity/manifest.py:249
msgid "Manifest file '{self.fh.base if self.fh else ''}' is corrupt: "
msgstr ""

#: ../duplicity/manifest.py:268
msgid "Manifests not equal because different volume numbers"
msgstr "Fichiers manifest non égaux car les numéros de volume diffèrent"

#: ../duplicity/manifest.py:273
msgid "Manifests not equal because volume lists differ"
msgstr "Fichiers manifest non égaux car les listes de volume diffèrent"

#: ../duplicity/manifest.py:277
msgid "Manifests not equal because hosts or directories differ"
msgstr "Fichiers manifest non égaux car les hôtes ou les répertoires diffèrent"

#: ../duplicity/manifest.py:438
msgid "Warning, found extra Volume identifier"
msgstr "Attention, un identifiant de volume supplémentaire a été trouvé"

#: ../duplicity/manifest.py:464
msgid "Other is not VolumeInfo"
msgstr "Autre n'est pas VolumeInfo"

#: ../duplicity/manifest.py:467
msgid "Volume numbers don't match"
msgstr "Les numéros de volume ne correspondent pas"

#: ../duplicity/manifest.py:470
msgid "start_indicies don't match"
msgstr "les indices_de_départ ne correspondent pas"

#: ../duplicity/manifest.py:473
msgid "end_index don't match"
msgstr "les indices_de_fin ne correspondent pas"

#: ../duplicity/manifest.py:478
msgid "Hashes don't match"
msgstr "Les empreintes ne correspondent pas"

#: ../duplicity/util.py:105 ../duplicity/dup_main.py:906
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "ERREUR_IGNORÉE : attention : ignore l'erreur comme demandé : %s : %s"

#: ../duplicity/util.py:170
#, python-format
msgid "Releasing lockfile %s"
msgstr "Libération du fichier de verrouillage %s"

#: ../duplicity/backends/pydrivebackend.py:203
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "Arrière-plan PyDrive : plusieurs fichiers nommés « %s »."

#: ../duplicity/backends/multibackend.py:100
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultiBackend : Impossible d'analyser la chaîne de requêtes %s : %s "

#: ../duplicity/backends/multibackend.py:110
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr "MultiBackend : Chaîne de requêtes %s non valide : plus d'une valeur pour %s"

#: ../duplicity/backends/multibackend.py:116
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:167
#: ../duplicity/backends/multibackend.py:174
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:186
#, python-format
msgid "MultiBackend: Url %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:189
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "MultiBackend : Impossible de charger le fichier de configuration %s : %s "

#: ../duplicity/backends/multibackend.py:196
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MultiBackend : utiliser le magasin de sauvegarde %s"

#: ../duplicity/backends/multibackend.py:200
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:214
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:257
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr ""

#: ../duplicity/backends/multibackend.py:272
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr "MultiBackend : échec de l'écriture dans le magasin de sauvegarde #%s (%s), essayez #%s, exception : %s"

#: ../duplicity/backends/multibackend.py:286
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr ""

#: ../duplicity/backends/multibackend.py:294
#, python-format
msgid "MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend : échec de l'écriture de %s. Essais faits dans tous les magasins de sauvegarde, aucun n'a fonctionné"

#: ../duplicity/backends/multibackend.py:315
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MutiBackend : impossible de mettre %s sur %s à partir de %s"

#: ../duplicity/backends/multibackend.py:320
#, python-format
msgid "MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr "MutiBackend : impossible d'obtenir %s. Essais faits dans tous les magasins de sauvegarde, aucun n'a fonctionné"

#: ../duplicity/backends/multibackend.py:330
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr ""

#: ../duplicity/backends/multibackend.py:334
msgid "Exception during list of {s.backend.parsed_url.strip_auth()}: "
msgstr ""

#: ../duplicity/backends/multibackend.py:342
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "MultiBackend : liste combinée : %s"

#: ../duplicity/backends/multibackend.py:371
#: ../duplicity/backends/multibackend.py:398
#, python-format
msgid "MultiBackend: failed to delete %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend : la supression de %s a échoué. Essais faits dans tous les magasins de sauvegarde, aucun n'a réussi"

#: ../duplicity/backends/giobackend.py:108
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Connexion échouée, veuillez vérifier votre mot de passe : %s"

#: ../duplicity/backends/onedrivebackend.py:90
#, python-format
msgid "Attempt of initialize_oauth2_session Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:67
msgid "Missing socket or ssl python modules."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:87
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:149
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:150
#, python-format
msgid "Using WebDAV directory %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:180
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:191
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:210
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:211
#: ../duplicity/backends/webdavbackend.py:234
#, python-format
msgid "WebDAV data length: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:214
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:220
#, python-format
msgid "WebDAV redirect to: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:222
msgid "WebDAV redirected 10 times. Giving up."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:227
msgid "WebDAV missing location header in redirect response."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:232
msgid "WebDAV retry request with authentification headers."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:233
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:237
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:256
msgid "python-kerberos needed to use kerberos                           authorization, falling back to basic auth."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:264
#, python-format
msgid "Kerberos authorization failed: %s.                          Falling back to basic auth."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:365
#, python-format
msgid "Creating missing directory %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:369
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:381
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:421
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:442
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:461
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/cli_data.py:102 ../duplicity/cli_data.py:162
#: ../duplicity/cli_data.py:196 ../duplicity/cli_data.py:243
#: ../duplicity/cli_data.py:346 ../duplicity/cli_data.py:364
#: ../duplicity/cli_data.py:524 ../duplicity/cli_data.py:706
#: ../duplicity/cli_data.py:932
msgid "path"
msgstr "chemin"

#: ../duplicity/cli_data.py:115
msgid "Hot|Cool|Archive"
msgstr ""

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/cli_data.py:120 ../duplicity/cli_data.py:126
#: ../duplicity/cli_data.py:134 ../duplicity/cli_data.py:168
#: ../duplicity/cli_data.py:432 ../duplicity/cli_data.py:448
#: ../duplicity/cli_data.py:495 ../duplicity/cli_data.py:512
#: ../duplicity/cli_data.py:518 ../duplicity/cli_data.py:536
#: ../duplicity/cli_data.py:602 ../duplicity/cli_data.py:610
#: ../duplicity/cli_data.py:662 ../duplicity/cli_data.py:743
#: ../duplicity/cli_data.py:919
msgid "number"
msgstr "nombre"

#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/cli_data.py:146 ../duplicity/cli_data.py:713
#: ../duplicity/cli_data.py:945
msgid "seconds"
msgstr "secondes"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/cli_data.py:184 ../duplicity/cli_data.py:190
#: ../duplicity/cli_data.py:358 ../duplicity/cli_data.py:669
#: ../duplicity/cli_data.py:911
msgid "gpg-key-id"
msgstr "identifiant_clé_gpg"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/cli_data.py:201 ../duplicity/cli_data.py:386
#: ../duplicity/cli_data.py:950
msgid "shell_pattern"
msgstr ""

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/cli_data.py:213 ../duplicity/cli_data.py:219
#: ../duplicity/cli_data.py:279 ../duplicity/cli_data.py:392
#: ../duplicity/cli_data.py:908
msgid "filename"
msgstr "nom_de_fichier"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/cli_data.py:225 ../duplicity/cli_data.py:549
#: ../duplicity/cli_data.py:976
msgid "time"
msgstr "temps"

#: ../duplicity/cli_data.py:237 ../duplicity/cli_data.py:398
msgid "regex"
msgstr ""

#: ../duplicity/cli_data.py:340
msgid "interval"
msgstr ""

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:352 ../duplicity/cli_data.py:506
#: ../duplicity/cli_data.py:555 ../duplicity/cli_data.py:685
#: ../duplicity/cli_data.py:923 ../duplicity/argparse311.py:1671
msgid "options"
msgstr ""

#: ../duplicity/cli_data.py:381
msgid "imap_mailbox"
msgstr "boîte_aux_lettres_IMAP"

#: ../duplicity/cli_data.py:406
msgid "If set, an extra file with statistics in json format will be stored with each backup. It includes the statistics printed out on stdout by default. With 'collection-status' it returs these stats."
msgstr ""

#: ../duplicity/cli_data.py:414
msgid "file_descriptor"
msgstr "descripteur_fichier"

#: ../duplicity/cli_data.py:421
msgid "log_filename"
msgstr ""

#: ../duplicity/cli_data.py:454
msgid "backup name"
msgstr "nom_sauvegarde"

#: ../duplicity/cli_data.py:561
msgid "s3_endpoint_url"
msgstr ""

#: ../duplicity/cli_data.py:629
msgid "s3_kms_key_id"
msgstr ""

#: ../duplicity/cli_data.py:635
msgid "s3_kms_grant"
msgstr ""

#: ../duplicity/cli_data.py:641
msgid "s3_region_name"
msgstr ""

#: ../duplicity/cli_data.py:647
msgid "policy"
msgstr ""

#. noun
#: ../duplicity/cli_data.py:652 ../duplicity/cli_data.py:657
#: ../duplicity/cli_data.py:897
msgid "command"
msgstr "commande"

#: ../duplicity/cli_data.py:692
msgid "pem formatted bundle of certificate authorities"
msgstr "Convertir au format PEM l'ensemble des autorités de certification"

#: ../duplicity/cli_data.py:697
msgid "path to a folder with certificate authority files"
msgstr ""

#. abbreviation for "character" (noun)
#: ../duplicity/cli_data.py:719 ../duplicity/cli_data.py:895
msgid "char"
msgstr "caract."

#: ../duplicity/cli_data.py:732
msgid "verb"
msgstr ""

#: ../duplicity/cli_data.py:750
msgid "extra headers for Webdav, like 'Cookie,name=value'"
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/cli_data.py:886
msgid "absolute_path"
msgstr "chemin_absolu"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/cli_data.py:889
msgid "alias"
msgstr ""

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:893
msgid "bucket_name"
msgstr "nom_de_bucket"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/cli_data.py:901
msgid "container_name"
msgstr "nom_de_conteneur"

#. noun
#: ../duplicity/cli_data.py:903
msgid "count"
msgstr "compte"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/cli_data.py:905
msgid "directory"
msgstr "dossier"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/cli_data.py:915
msgid "module"
msgstr ""

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:926
msgid "other.host"
msgstr "autre.hote"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:929
msgid "password"
msgstr "mot_de_passe"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:935
msgid "port"
msgstr ""

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:939
msgid "prefix"
msgstr "préfixe"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/cli_data.py:942
msgid "relative_path"
msgstr "chemin_relatif"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/cli_data.py:954
msgid "some_dir"
msgstr "dossier_quelconque"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:959
msgid "source_path"
msgstr ""

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:963
msgid "source_url"
msgstr "url_source"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:968
msgid "target_dir"
msgstr "dossier_cible"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:972
msgid "target_url"
msgstr "url_cible"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:980
msgid "user"
msgstr "utilisateur"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/cli_data.py:982
msgid "account_id"
msgstr "identifiant_de_compte"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/cli_data.py:985
msgid "application_key"
msgstr "clef_d_application"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/cli_data.py:988 ../duplicity/dup_collections.py:565
msgid "remote"
msgstr "distant"

#: ../duplicity/cli_data.py:992
msgid "Backends and their URL formats:"
msgstr "Moteurs et leurs formats URL :"

#: ../duplicity/diffdir.py:102 ../duplicity/diffdir.py:401
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Erreur %s pour l'obtention du delta pour %s"

#: ../duplicity/diffdir.py:116
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Obtention du delta de %s et %s"

#: ../duplicity/diffdir.py:160
#, python-format
msgid "A %s"
msgstr ""

#: ../duplicity/diffdir.py:168
#, python-format
msgid "M %s"
msgstr ""

#: ../duplicity/diffdir.py:191
#, python-format
msgid "Comparing %s and %s"
msgstr "Comparaison de %s et %s"

#: ../duplicity/diffdir.py:204
#, python-format
msgid "D %s"
msgstr ""

#: ../duplicity/librsync.py:194
msgid "basis_file must be a (true) file or an object whose file attribute is the underlying true file object"
msgstr ""

#: ../duplicity/dup_time.py:54
#, python-format
msgid "Bad interval string \"%s\"\n\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr "Mauvais chaîne d'intervalle « %s »\n\n"
"Les intervalles sont spécifiés comme 2Y (2 ans) ou 2h30min (2,5 heures). Les\n"
"caractères spéciaux autorisés sont s, m, h, D, W, M et Y. Voir la page de manuel\n"
"pour plus d'informations."

#: ../duplicity/dup_time.py:63
#, python-format
msgid "Bad time string \"%s\"\n\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr "Mauvais chaîne de temps « %s »\n\n"
"Les chaînes de temps acceptées sont les intervalles (comme « 3D64s »), des chaînes\n"
"au format w3-datetime, comme « 2002-04-26T04:22:01-07:00 » (les chaînes comme\n"
"« 2002-04-26T04:22:01 » sont aussi acceptées - duplicity utilisera le \n"
"fuseau horaire courant), ou des dates ordinaires comme 2/4/1997 ou 2001-04-23\n"
"(plusieurs combinaisons sont possibles, mais le mois précède toujours le jour)."

#: ../duplicity/cli_main.py:155
msgid "No valid action found. Will imply 'backup' because a path source was given and target is a url location."
msgstr ""

#: ../duplicity/cli_main.py:163
msgid "No valid action found. Will imply 'restore' because url source was given and target is a local path."
msgstr ""

#: ../duplicity/cli_main.py:170
#, python-brace-format
msgid "Invalid '{remainder[0]}' action and cannot be implied from the "
msgstr ""

#: ../duplicity/cli_main.py:291
#, python-brace-format
msgid "GPG binary is {config.gpg_binary}, version {gpg_version}"
msgstr ""

#: ../duplicity/cli_main.py:327
#, python-brace-format
msgid "Using archive dir: {config.archive_dir_path.uc_name}"
msgstr ""

#: ../duplicity/cli_main.py:328
#, python-brace-format
msgid "Using backup name: {config.backup_name}"
msgstr ""

#: ../duplicity/dup_main.py:113
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Réutiliser la PASSPHRASE configurée comme SIGN_PASSPHRASE"

#: ../duplicity/dup_main.py:124
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Réutiliser la SIGN_PASSPHRASE configurée comme PASSPHRASE"

#: ../duplicity/dup_main.py:168
msgid "PASSPHRASE variable not set, asking user."
msgstr "Variable PASSPHRASE non établie, demande à l'utilisateur."

#: ../duplicity/dup_main.py:193
msgid "Retype passphrase for signing key to confirm: "
msgstr "Veuillez retaper la phrase de passe pour la clé de signature pour confirmer : "

#: ../duplicity/dup_main.py:195
msgid "Retype passphrase for decryption to confirm: "
msgstr ""

#: ../duplicity/dup_main.py:199
msgid "First and second passphrases do not match!  Please try again."
msgstr "La première et la seconde phrases de passe ne correspondent pas ! Veuillez réessayer."

#: ../duplicity/dup_main.py:210
msgid "Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr "Impossible d'utiliser une phrase de passe vide avec un chiffrement symétrique ! Veuillez réessayer."

#: ../duplicity/dup_main.py:270
#, python-format
msgid "File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr "Le fichier %s est terminé dans le jeu de sauvegarde.\n"
"Le redémarrage continue sur fichier %s."

#: ../duplicity/dup_main.py:280
#, python-format
msgid "File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr "Le fichier %s est manquant dans le jeu de sauvegarde.\n"
"Le redémarrage continue sur le fichier %s."

#: ../duplicity/dup_main.py:335
#, python-format
msgid "%s Remote filesize %d for %s does not match local size %d, retrying."
msgstr ""

#: ../duplicity/dup_main.py:342
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Le fichier % s a été endommagé pendant le téléversement."

#: ../duplicity/dup_main.py:382
msgid "Restarting backup, but current encryption settings do not match original settings"
msgstr "Redémarrage de sauvegarde, mais les paramètres de chiffrement actuels ne correspondent pas aux paramètres originaux"

#: ../duplicity/dup_main.py:404
msgid "Skipping encryption validation due to glacier/deep storage"
msgstr ""

#: ../duplicity/dup_main.py:408
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Redémarrage après le volume %s, le fichier %s, le bloc %s"

#: ../duplicity/dup_main.py:471
msgid "Skipped volume upload, as effectivly nothing has changed"
msgstr ""

#: ../duplicity/dup_main.py:485
#, python-format
msgid "Processed volume %d"
msgstr "Volume %d traité"

#: ../duplicity/dup_main.py:653
msgid "Fatal Error: Unable to start incremental backup.  Old signatures not found and incremental specified"
msgstr "Erreur fatale : impossible de démarrer la sauvegarde incrémentale. Les anciennes signatures sont introuvables pour la méthode incrémentale indiquée"

#: ../duplicity/dup_main.py:659
msgid "No signatures found, switching to full backup."
msgstr "Aucune signature de sauvegarde trouvée, bascule vers une sauvegarde complète."

#: ../duplicity/dup_main.py:673
msgid "Backup Statistics"
msgstr "Statistiques de sauvegarde"

#: ../duplicity/dup_main.py:813
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "%s introuvable dans l'archive - aucun fichier n'a été restauré."

#: ../duplicity/dup_main.py:817
msgid "No files found in archive - nothing restored."
msgstr "Aucun fichier trouvé dans l’archive – rien n'a été restauré."

#: ../duplicity/dup_main.py:853
#, python-format
msgid "Processed volume %d of %d"
msgstr "Le volume %d sur %d a été traité"

#: ../duplicity/dup_main.py:896
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Données non valides - empreinte %s non vérifiée pour le fichier :"

#: ../duplicity/dup_main.py:898
#, python-format
msgid "Calculated hash: %s"
msgstr "Empreinte calculée : %s"

#: ../duplicity/dup_main.py:899
#, python-format
msgid "Manifest hash: %s"
msgstr "Empreinte du fichier manifest : %s"

#: ../duplicity/dup_main.py:952
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Le volume a été signé par la clé %s et non pas %s"

#: ../duplicity/dup_main.py:984
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Vérification complète : %s, %s."

#: ../duplicity/dup_main.py:985
#, python-format
msgid "%d file(s) compared"
msgstr ""

#: ../duplicity/dup_main.py:985
#, python-format
msgid "%d difference(s) found"
msgstr ""

#: ../duplicity/dup_main.py:1004
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Aucun fichier étranger détecté, aucune suppression effectuée lors du nettoyage."

#: ../duplicity/dup_main.py:1019
msgid "Found the following file(s) to delete:"
msgstr ""

#: ../duplicity/dup_main.py:1023
msgid "Run duplicity again with the --force option to actually delete."
msgstr "Exécutez duplicity à nouveau avec l’option --force pour supprimer réellement."

#: ../duplicity/dup_main.py:1069
msgid "There are backup set(s) at time(s):"
msgstr "Il y a un ou plusieurs jeux de sauvegarde datés du :"

#: ../duplicity/dup_main.py:1071
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Qui ne peuvent être supprimés car de nouveaux jeux en dépendent."

#: ../duplicity/dup_main.py:1078
msgid "Current active backup chain is older than specified time.  However, it will not be deleted.  To remove all your backups, manually purge the repository."
msgstr "La chaîne de sauvegarde active actuelle est plus ancienne que la date indiquée. Cependant elle ne sera pas supprimée. Pour ôter toutes vos sauvegardes, purgez manuellement leur dépôt."

#: ../duplicity/dup_main.py:1096
msgid "No old backup sets found, nothing deleted."
msgstr "Aucun ancien jeu de sauvegarde n’a été trouvé, rien n’a été supprimé."

#: ../duplicity/dup_main.py:1099
msgid "Deleting backup chain(s) at time:"
msgstr ""

#: ../duplicity/dup_main.py:1108
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr ""

#: ../duplicity/dup_main.py:1110
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr ""

#: ../duplicity/dup_main.py:1113
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Suppression de la chaîne de signature complète %s"

#: ../duplicity/dup_main.py:1115
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Suppression de la chaîne de sauvegarde complète %s"

#: ../duplicity/dup_main.py:1122
msgid "Found old backup chain(s) at the following time:"
msgstr ""

#: ../duplicity/dup_main.py:1126
msgid "Rerun command with --force option to actually delete."
msgstr "Veuillez relancer la commande avec l'option --force pour forcer la suppression."

#: ../duplicity/dup_main.py:1229
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Supprime le %s local (ne fait pas autorité au niveau du serveur)."

#: ../duplicity/dup_main.py:1233
#, python-format
msgid "Unable to delete %s: %s"
msgstr "Impossible de supprimer %s : %s"

#: ../duplicity/dup_main.py:1267 ../duplicity/dup_temp.py:296
#, python-format
msgid "Failed to read %s: %s"
msgstr "Impossible de lire %s : %s"

#: ../duplicity/dup_main.py:1279
#, python-format
msgid "Copying %s to local cache."
msgstr "Copie de %s vers le cache local."

#: ../duplicity/dup_main.py:1327
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr "Les métadonnées locales et distantes sont déjà synchronisées. Aucune synchronisation nécessaire."

#: ../duplicity/dup_main.py:1332
msgid "Synchronizing remote metadata to local cache..."
msgstr "Synchronisation des métadonnées distantes vers le cache local..."

#: ../duplicity/dup_main.py:1346
msgid "Sync would copy the following from remote to local:"
msgstr "La synchronisation va copier localement les éléments suivants à partir de l’hôte distant :"

#: ../duplicity/dup_main.py:1352
msgid "Sync would remove the following spurious local files:"
msgstr "La synchronisation va supprimer les fichiers parasites suivants :"

#: ../duplicity/dup_main.py:1398
msgid "Unable to get free space on temp."
msgstr "Impossible de déterminer l’espace libre dans le dossier temporaire."

#: ../duplicity/dup_main.py:1405
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr "Il y a %d d’espace temporaire disponible, la sauvegarde nécessite environ %d."

#: ../duplicity/dup_main.py:1409
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr "Il y a %d d’espace temporaire disponible, la sauvegarde va nécessiter environ %d."

#: ../duplicity/dup_main.py:1416
msgid "Unable to get max open files."
msgstr "Impossible d’obtenir le maximum de fichiers ouverts."

#: ../duplicity/dup_main.py:1421
#, python-format
msgid "Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr "Le nombre maximum de fichiers ouverts pour %s est trop bas, il devrait être >= 1024.\n"
"Utilisez « ulimit -n 1024 » (ou supérieur) pour corriger.\n"

#: ../duplicity/dup_main.py:1477
msgid "RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr "REDÉMARRAGE : échec de téléchargement du premier volume avant le délai de fin.\n"
"              Le redémarrage est impossible... lancement de la sauvegarde depuis le début."

#: ../duplicity/dup_main.py:1487
#, python-format
msgid "RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr "REDÉMARRAGE : échec de téléchargement de %d volumes sur %d avant le délai de fin.\n"
"              Redémarrage de la sauvegarde au volume %d."

#: ../duplicity/dup_main.py:1498
#, python-format
msgid "RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last partial\n"
"         backup then restart the backup from the beginning."
msgstr "REDÉMARRAGE : état de sauvegarde impossible :\n"
"              le fichier manifest indique %d volumes, le distant en a %d.\n"
"              Redémarrage impossible... duplicity va nettoyer la dernière\n"
"              sauvegarde partielle puis reprendre la sauvegarde depuis le début."

#: ../duplicity/dup_main.py:1523
msgid "PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr ""

#: ../duplicity/dup_main.py:1549
#, python-format
msgid "Acquiring lockfile %s"
msgstr ""

#: ../duplicity/dup_main.py:1609
#, python-brace-format
msgid "Last {action} backup left a partial set, restarting."
msgstr ""

#: ../duplicity/dup_main.py:1613
#, python-brace-format
msgid "Cleaning up previous partial {action} backup set, restarting."
msgstr ""

#: ../duplicity/dup_main.py:1627
msgid "Last full backup date: none"
msgstr "Date de la dernière sauvegarde complète : aucune"

#: ../duplicity/dup_main.py:1634
msgid "Last full backup is too old, forcing full backup"
msgstr "La dernière sauvegarde complète est trop ancienne, forçage d’une nouvelle sauvegarde complète"

#: ../duplicity/dup_main.py:1690
msgid "When using symmetric encryption, the signing passphrase must equal the encryption passphrase."
msgstr "Lors de l'utilisation du chiffrement symétrique, le mot de passe de signature doit correspondre au mot de passe de chiffrement."

#: ../duplicity/asyncscheduler.py:259
#, python-format
msgid "task execution done (success: %s)"
msgstr "exécution de tâche réalisée (succès : %s)"

#: ../duplicity/gpginterface.py:694
#, python-brace-format
msgid "GPG process {process.pid} terminated before wait()"
msgstr ""

#: ../duplicity/robust.py:84
#, python-format
msgid "Error listing directory %s"
msgstr "Erreur de listage du dossier %s"

#: ../duplicity/dup_collections.py:200 ../duplicity/dup_collections.py:209
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "BackupSet.delete : %s manquant"

#: ../duplicity/dup_collections.py:235
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Erreur fatale : aucun fichier manifest présent pour la sauvegarde la plus récente"

#: ../duplicity/dup_collections.py:248
msgid "Fatal Error: Remote manifest does not match local one.  Either the remote backup set or the local archive directory has been corrupted."
msgstr "Erreur fatale : le fichier manifest distant ne correspond pas à celui en local. Soit le jeu de sauvegarde distant, soit le dossier local d’archive a été corrompu."

#: ../duplicity/dup_collections.py:256
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr ""

#: ../duplicity/dup_collections.py:265
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr "Erreur fatale : ni le fichier manifest local, ni le distant ne sont lisibles."

#: ../duplicity/dup_collections.py:276
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "Traitement du manifeste local %s (%s)"

#: ../duplicity/dup_collections.py:291
msgid "Error processing remote file ({os.fsdecode(remote_file)}): {util.uexc(message)}"
msgstr ""

#: ../duplicity/dup_collections.py:293
msgid "Processing remote file {os.fsdecode(remote_file)} ({len(remote_file_buffer)})"
msgstr ""

#: ../duplicity/dup_collections.py:311
msgid "No Jsonstat file found, return enmty."
msgstr ""

#: ../duplicity/dup_collections.py:404
msgid "Preferring Backupset over previous one!"
msgstr "Jeu de sauvegarde préféré au précédent !"

#: ../duplicity/dup_collections.py:408
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr "Ignore le jeu de sauvegarde incrémentale (date_début : %s ; nécessaire : %s)"

#: ../duplicity/dup_collections.py:417
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr "Jeu de sauvegarde incrémentale ajouté (date_début : %s ; date_fin : %s)"

#: ../duplicity/dup_collections.py:490
msgid "Chain start time: "
msgstr "Date de début de chaîne : "

#: ../duplicity/dup_collections.py:491
msgid "Chain end time: "
msgstr "Date de fin de chaîne : "

#: ../duplicity/dup_collections.py:492
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Nombre de jeux de sauvegarde contenus : %d"

#: ../duplicity/dup_collections.py:493
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Nombre total de volumes contenus : %d"

#: ../duplicity/dup_collections.py:494 ../duplicity/dup_collections.py:1308
msgid "Type of backup set:"
msgstr "Type de jeu de sauvegarde :"

#: ../duplicity/dup_collections.py:494 ../duplicity/dup_collections.py:1308
msgid "Time:"
msgstr "Date :"

#: ../duplicity/dup_collections.py:494
msgid "Num volumes:"
msgstr "Nombre de volumes :"

#: ../duplicity/dup_collections.py:499 ../duplicity/dup_collections.py:1315
msgid "Full"
msgstr "Complète"

#: ../duplicity/dup_collections.py:502 ../duplicity/dup_collections.py:1317
msgid "Incremental"
msgstr "Incrémentale"

#: ../duplicity/dup_collections.py:563
msgid "local"
msgstr ""

#: ../duplicity/dup_collections.py:726
msgid "Collection Status"
msgstr "État de la collection"

#: ../duplicity/dup_collections.py:728
#, python-format
msgid "Connecting with backend: %s"
msgstr "Connexion au serveur central : %s"

#: ../duplicity/dup_collections.py:729
#, python-format
msgid "Archive dir: %s"
msgstr "Dossier d’archive : %s"

#: ../duplicity/dup_collections.py:732
#, python-format
msgid "Found %d secondary backup chain(s)."
msgstr ""

#: ../duplicity/dup_collections.py:734
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Chaîne secondaire %d sur %d :"

#: ../duplicity/dup_collections.py:739
msgid "Found primary backup chain with matching signature chain:"
msgstr "Chaîne primaire de sauvegarde trouvée avec la signature de chaîne correspondante :"

#: ../duplicity/dup_collections.py:742
msgid "No backup chains with active signatures found"
msgstr "Aucune chaîne de sauvegarde avec les signatures actives n’a été trouvée"

#: ../duplicity/dup_collections.py:745
#, python-format
msgid "Also found %d backup set(s) not part of any chain,"
msgstr ""

#: ../duplicity/dup_collections.py:746
#, python-format
msgid "and %d incomplete backup set(s)."
msgstr ""

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:748
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr "Ceux-ci peuvent être supprimés en exécutant duplicity avec la commande « cleanup »."

#: ../duplicity/dup_collections.py:750
msgid "No orphaned or incomplete backup sets found."
msgstr "Aucun jeu orphelin ou incomplet de sauvegarde n’a été trouvé."

#: ../duplicity/dup_collections.py:766
#, python-format
msgid "%d file(s) exists on backend"
msgstr ""

#: ../duplicity/dup_collections.py:772
#, python-format
msgid "%d file exists in cache"
msgid_plural "%d files exist in cache"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/dup_collections.py:831
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr "Avertissement, rejet du dernier jeu de sauvegarde à cause de l’absence du fichier de signatures."

#: ../duplicity/dup_collections.py:854
msgid "Warning, found the following local orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:862
msgid "Warning, found the following remote orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:870
msgid "Warning, found signatures but no corresponding backup files"
msgstr "Avertissement, signatures trouvées mais aucun fichier de sauvegarde correspondant"

#: ../duplicity/dup_collections.py:876
msgid "Warning, found incomplete backup sets, probably left from aborted session"
msgstr "Avertissement, jeux de sauvegarde incomplets trouvés, probablement laissés par des sessions interrompues"

#: ../duplicity/dup_collections.py:882
msgid "Warning, found the following orphaned backup file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:897
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Extraction des chaînes de sauvegarde depuis la liste des fichiers : %s"

#: ../duplicity/dup_collections.py:908
#, python-format
msgid "File %s is part of known set"
msgstr "Le fichier %s fait partie d’un jeu connu"

#: ../duplicity/dup_collections.py:911
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "Le fichier %s ne fait pas partie d’un jeu connu ; création d’un nouveau jeu"

#: ../duplicity/dup_collections.py:916
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Fichier ignoré (rejeté par le jeu de sauvegarde) « %s »"

#: ../duplicity/dup_collections.py:932
#, python-format
msgid "Found backup chain %s"
msgstr "Chaîne de sauvegarde %s trouvée"

#: ../duplicity/dup_collections.py:937
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Jeu %s ajouté à la chaîne préexistante %s"

#: ../duplicity/dup_collections.py:940
#, python-format
msgid "Found orphaned set %s"
msgstr "Jeu orphelin %s trouvé"

#: ../duplicity/dup_collections.py:1088
#, python-format
msgid "No signature chain for the requested time. Using oldest available chain, starting at time %s."
msgstr ""

#: ../duplicity/dup_collections.py:1306
#, python-format
msgid "File: %s"
msgstr "Fichier : %s"

#: ../duplicity/dup_collections.py:1307
#, python-format
msgid "Total number of backup: %d"
msgstr "Nombre total de sauvegardes : %d"

#: ../duplicity/dup_collections.py:1308 ../duplicity/dup_collections.py:1344
msgid "Type of file change:"
msgstr "Type de changement de fichier :"

#: ../duplicity/dup_collections.py:1342
#, python-format
msgid " Backup set time: %s"
msgstr ""

#: ../duplicity/dup_collections.py:1343
#, python-format
msgid "Total number of changes: %d"
msgstr ""

#: ../duplicity/dup_collections.py:1344
msgid "File:"
msgstr ""

#: ../duplicity/patchdir.py:72 ../duplicity/patchdir.py:79
#, python-format
msgid "Patching %s"
msgstr "En train de réparer %s"

#: ../duplicity/patchdir.py:547
#, python-format
msgid "Error '%s' patching %s"
msgstr "Erreur « %s » pendant la réparation de %s"

#: ../duplicity/patchdir.py:624
#, python-format
msgid "Writing %s of type %s"
msgstr "Écriture %s de type %s"

#: ../duplicity/selection.py:115
#, python-format
msgid "Skipping socket %s"
msgstr "Passer le socket %s"

#: ../duplicity/selection.py:121
#, python-format
msgid "Error initializing file %s"
msgstr "Erreur d'initialisation du fichier %s"

#: ../duplicity/selection.py:127 ../duplicity/selection.py:167
#: ../duplicity/selection.py:615
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Erreur d'accès au fichier %s éventuellement verrouillé"

#: ../duplicity/selection.py:182
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Attention : la base %s n'existe pas, continue"

#: ../duplicity/selection.py:184 ../duplicity/selection.py:205
#: ../duplicity/selection.py:208
#, python-format
msgid "Selecting %s"
msgstr "Sélection %s"

#: ../duplicity/selection.py:263
#, python-format
msgid "                The last file selection option is the filter option %s, which will have no\n"
"                effect as there are no subsequent file selection options. Exiting because\n"
"                this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:278
#, python-format
msgid "                Only these filter mode options were specified:\n"
"                    %s\n"
"                Case sensitive globbing is the default behaviour and so this has no effect.\n"
"                Exiting because this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:346
#, python-format
msgid "                Fatal Error: The file specification\n"
"                    %s\n"
"                cannot match any files in the base directory\n"
"                    %s\n"
"                Useful file specifications begin with the base directory or some\n"
"                pattern (such as '**') which matches the base directory."
msgstr ""

#: ../duplicity/selection.py:360
#, python-format
msgid "Fatal Error while processing expression\n"
"%s"
msgstr "Erreur fatale pendant le traitement de l'expression\n"
"%s"

#: ../duplicity/selection.py:378
#, python-format
msgid "Reading files to backup from list %s"
msgstr ""

#: ../duplicity/selection.py:403
#, python-format
msgid "                Files-from list contains the absolute path:\n"
"                    %s\n"
"                All paths specified in a files-from list must be given relative to the backup\n"
"                source path."
msgstr ""

#: ../duplicity/selection.py:418
msgid "                Files-from list specified which contains no files, the backup will be empty as\n"
"                a result. Exiting as this probably isn't what you meant,"
msgstr ""

#: ../duplicity/selection.py:435
#, python-format
msgid "                Last selection expression:\n"
"                    %s\n"
"                only specifies that files be included.  Because the default is to\n"
"                include all files, the expression is redundant.  Exiting because this\n"
"                probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:497
#, python-format
msgid "Reading %s filelist %s"
msgstr ""

#: ../duplicity/selection.py:538
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Erreur en compilant l'expression régulière %s"

#: ../duplicity/lazy.py:349
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Attention : ancienindice %s >= nouvelindice %s"

#: ../duplicity/lazy.py:424
#, python-format
msgid "Error '%s' processing %s"
msgstr "Erreur « %s » lors du traitement de %s"

#: ../duplicity/lazy.py:436
#, python-format
msgid "Skipping %s because of previous error"
msgstr "Passe %s à cause d'une erreur précédente"

#: ../duplicity/path.py:109
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr "Avertissement : %s nombres de développement non valides (0x%X), considérés comme étant (0, 0)."

#: ../duplicity/path.py:239 ../duplicity/path.py:299
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Attention : %s a un mtime négatif, il sera traité comme étant 0."

#: ../duplicity/path.py:372
#, python-format
msgid "New file %s"
msgstr "Nouveau fichier %s"

#: ../duplicity/path.py:375
#, python-format
msgid "File %s is missing"
msgstr "Fichier %s manquant"

#: ../duplicity/path.py:378
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Le fichier %%s est de type %s, %s attendu"

#: ../duplicity/path.py:383 ../duplicity/path.py:412
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Le fichier %%s a les permissions %s, %s attendues"

#: ../duplicity/path.py:389
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Le fichier %%s a un mtime %s, %s attendu"

#: ../duplicity/path.py:400
#, python-format
msgid "Data for file %s is different"
msgstr "les données pour le fichier %s sont différentes"

#: ../duplicity/path.py:408
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "Le lien symbolique %%s pointe vers %s, %s attendu"

#: ../duplicity/path.py:415
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Le fichier de périphérique %%s a le numéro %s, %s attendu"

#: ../duplicity/path.py:605
#, python-format
msgid "Making directory %s"
msgstr "Création du dossier %s"

#: ../duplicity/path.py:615
#, python-format
msgid "Deleting %s"
msgstr "Suppression de %s"

#: ../duplicity/path.py:624
#, python-format
msgid "Touching %s"
msgstr "Touche %s"

#: ../duplicity/path.py:632
#, python-format
msgid "Deleting tree %s"
msgstr "Suppression de l'arbre %s"

#: ../duplicity/cli_util.py:48
msgid "Enter 'duplicity --help' for help screen."
msgstr ""

#: ../duplicity/cli_util.py:127
msgid "Running in 'ignore errors' mode due to --ignore-errors.\n"
"Please reconsider if this was not intended"
msgstr ""

#: ../duplicity/cli_util.py:139
msgid "Use of the --asynchronous-upload option is experimental and not safe for production! There are reported cases of undetected data loss during upload. Be aware and periodically verify your backups to be safe."
msgstr ""

#: ../duplicity/cli_util.py:152
msgid "'{val}' is not an int: {str(e)}"
msgstr ""

#: ../duplicity/cli_util.py:166
#, python-brace-format
msgid "'{val} is not a single character."
msgstr ""

#: ../duplicity/cli_util.py:193
#, python-brace-format
msgid "Source should be pathname, not url.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:195
#, python-brace-format
msgid "Argument source_path '{val}' does not exist."
msgstr ""

#: ../duplicity/cli_util.py:201
#, python-brace-format
msgid "Source should be url, not directory.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:207
#, python-brace-format
msgid "Target should be directory, not url.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:212
msgid "Unable to create target dir '{val}': {str(e)}"
msgstr ""

#: ../duplicity/cli_util.py:218
#, python-brace-format
msgid "Target should be url, not directory.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:263
msgid "Verbosity must be one of: digit [0-9], character [ewnid],\n"
"or word ['error', 'warning', 'notice', 'info', 'debug'].\n"
"The default is 3 (Notice).  It is strongly recommended\n"
"that verbosity level is set at 2 (Warning) or higher."
msgstr ""

#: ../duplicity/cli_util.py:399
#, python-brace-format
msgid "Specified archive directory '{archive_dir_path.uc_name}' is not a directory"
msgstr ""

#: ../duplicity/cli_util.py:408
msgid "Encrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/cli_util.py:428
msgid "Hidden dncrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/cli_util.py:442
msgid "Sign key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/argparse311.py:291
msgid "usage: "
msgstr ""

#: ../duplicity/argparse311.py:761
#, python-format
msgid "argument %(argument_name)s: %(message)s"
msgstr ""

#: ../duplicity/argparse311.py:870
msgid ".__call__() not defined"
msgstr ""

#: ../duplicity/argparse311.py:1115
#, python-format
msgid "conflicting subparser: %s"
msgstr ""

#: ../duplicity/argparse311.py:1118
#, python-format
msgid "conflicting subparser alias: %s"
msgstr ""

#: ../duplicity/argparse311.py:1152
#, python-format
msgid "unknown parser %(parser_name)r (choices: %(choices)s)"
msgstr ""

#: ../duplicity/argparse311.py:1215
#, python-format
msgid "argument \"-\" with mode %r"
msgstr ""

#: ../duplicity/argparse311.py:1223
#, python-format
msgid "can't open '%(filename)s': %(error)s"
msgstr ""

#: ../duplicity/argparse311.py:1427
#, python-format
msgid "cannot merge actions - two groups are named %r"
msgstr ""

#: ../duplicity/argparse311.py:1462
msgid "'required' is an invalid argument for positionals"
msgstr ""

#: ../duplicity/argparse311.py:1483
#, python-format
msgid "invalid option string %(option)r: must start with a character %(prefix_chars)r"
msgstr ""

#: ../duplicity/argparse311.py:1500
#, python-format
msgid "dest= is required for options like %r"
msgstr ""

#: ../duplicity/argparse311.py:1517
#, python-format
msgid "invalid conflict_resolution value: %r"
msgstr ""

#: ../duplicity/argparse311.py:1534
#, python-format
msgid "conflicting option string: %s"
msgid_plural "conflicting option strings: %s"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/argparse311.py:1595
msgid "mutually exclusive arguments must be optional"
msgstr ""

#: ../duplicity/argparse311.py:1670
msgid "positional arguments"
msgstr ""

#: ../duplicity/argparse311.py:1689
msgid "show this help message and exit"
msgstr ""

#: ../duplicity/argparse311.py:1721
msgid "cannot have multiple subparser arguments"
msgstr ""

#: ../duplicity/argparse311.py:1769 ../duplicity/argparse311.py:2264
#, python-format
msgid "unrecognized arguments: %s"
msgstr ""

#: ../duplicity/argparse311.py:1868
#, python-format
msgid "not allowed with argument %s"
msgstr ""

#: ../duplicity/argparse311.py:1912 ../duplicity/argparse311.py:1926
#, python-format
msgid "ignored explicit argument %r"
msgstr ""

#: ../duplicity/argparse311.py:2030
#, python-format
msgid "the following arguments are required: %s"
msgstr ""

#: ../duplicity/argparse311.py:2042
#, python-format
msgid "one of the arguments %s is required"
msgstr ""

#: ../duplicity/argparse311.py:2083
msgid "expected one argument"
msgstr ""

#: ../duplicity/argparse311.py:2084
msgid "expected at most one argument"
msgstr ""

#: ../duplicity/argparse311.py:2085
msgid "expected at least one argument"
msgstr ""

#: ../duplicity/argparse311.py:2089
#, python-format
msgid "expected %s argument"
msgid_plural "expected %s arguments"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/argparse311.py:2143
#, python-format
msgid "ambiguous option: %(option)s could match %(matches)s"
msgstr ""

#: ../duplicity/argparse311.py:2207
#, python-format
msgid "unexpected option string: %s"
msgstr ""

#: ../duplicity/argparse311.py:2402
#, python-format
msgid "%r is not callable"
msgstr ""

#: ../duplicity/argparse311.py:2419
#, python-format
msgid "invalid %(type)s value: %(value)r"
msgstr ""

#: ../duplicity/argparse311.py:2429
#, python-format
msgid "invalid choice: %(value)r (choose from %(choices)s)"
msgstr ""

#: ../duplicity/argparse311.py:2505
#, python-format
msgid "%(prog)s: error: %(message)s\n"
msgstr ""

